rpc/server_auth.go

```diff
@@ -239,16 +241,16 @@ func (wrapped ctxWrappedServerStream) Context() context.Context {
 func tokenFromContext(ctx context.Context) (string, error) {
 	md, ok := metadata.FromIncomingContext(ctx)
 	if !ok {
-		return "", status.Error(codes.Unauthenticated, "authentication required")
+		return "", status.Error(codes.Unauthenticated, "authentication required no md")
 	}
-	authHeader := md.Get(metadataFieldAuthorization)
+	authHeader := md.Get(MetadataFieldAuthorization)
 	if len(authHeader) != 1 {
 		return "", status.Error(codes.Unauthenticated, "authentication required")
 	}
-	if !strings.HasPrefix(authHeader[0], authorizationValuePrefixBearer) {
-		return "", status.Errorf(codes.Unauthenticated, "expected Authorization: %s", authorizationValuePrefixBearer)
+	if !strings.HasPrefix(authHeader[0], AuthorizationValuePrefixBearer) {
+		return "", status.Errorf(codes.Unauthenticated, "expected Authorization: %s", AuthorizationValuePrefixBearer)
```

Comment:
[minor] this is such a bizarre error message?

Reply:
Agreed! I updated it to be more specific.